app/app/fixtures/grants.json

```diff
@@ -1,265 +1,259 @@
 [
```

Comment:
Why are there so many changes to the fixtures?

Reply:
Looks like the changes are mostly formatting changes. I think what happened is when I did a find + replace to update the Rinkeby DAI address to use the official Maker address in 8330a2076ee4cd72ae15acd60df95dc8dcb2bac8, VSCode must have auto-formatted these JSON files based on my default VSCode settings. Looks like indents were updated from 4 spaces to 2, along with other formatting changes. 

Oddly, I don't recall seeing any formatting changes when looking at the original diff, and @apbendi would have noticed these during the original PR, so I'm totally sure how these changes got in. I'm guessing they snuck in during a rebase somehow.

Reply:
Ok, thanks for the explanation - always good to limit the diffs to the scope so that its easier to review and see where the meaningful changes are